clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -131,6 +117,24 @@ def start_clipper(
             logger.warning("Error starting Clipper: {}".format(e.msg))
             raise e
 
+        def parse_configuration(file):
+            if(file):
+                with open(file, "r") as stream:
+                    config = yaml.load(stream)
+                    for name in config:
+                        obj_type = config[name]["obj_type"]
+                        input_type = config[name]["input_type"]
+                        if(obj_type == "application"):
+                            slo = config[name]["slo"]
+                            default_value = config[name]["default_value"]
+                            self.register_application(name, input_type, default_value, slo)
+                        elif(obj_type == "model"):
```

Comment:
Let's add support for specifying parameter values for the `deploy_model` function that have default values (`labels`, `num_replicas`). Any additional key-word arguments should be passed via the `**kwargs` parameter.